### PR TITLE
translate to chinese zh-CN

### PR DESCRIPTION
**Pull request by zhizhongzhiwai**
I am a developer from China

This submission fixes some of the wrong translations, while making the translation more in line with the Chinese mainland's reading habits

### CODE REVIEW

File: lang/languages.xml [hunk 1]
```diff
@@ -29,6 +29,6 @@ List of available languages for your wallet's seed:
         <!-- <language display_name="Polski" locale="pl_PL" wallet_language="English" flag="/lang/flags/poland.png" qs="none"/> -->
         <language display_name="Português (BR)" locale="pt-br_BR" wallet_language="Portuguese" flag="/lang/flags/brazil.png" qs="none"/>
         <!-- <language display_name="русский язык" locale="ru_RU" wallet_language="Russian" flag="/lang/flags/russia.png" qs="none"/> -->
-        <language display_name="简体中文 (中国)" locale="zh-cn_CN" wallet_language="English" flag="/lang/flags/china.png" qs="none"/>
+        <language display_name="简体中文 (中国)" locale="zh-cn_CN" wallet_language="Chinese" flag="/lang/flags/china.png" qs="none"/>
```
Comment (NanoAkron) [inline lang/languages.xml]: Making the wallet language `Chinese` requires a simplified Chinese word list of 1626 characters. You can use the one at `https://github.com/spesmilo/electrum/tree/master/lib/wordlist` to make one for Monero.

**Comment by zhizhongzhiwai** [comment]
#671  has been fixed.

I spent two days trying to fix all the wrong translations. Including #671 

This translation inspect bitcoin wallet multibit and electrum.  the same function , the same translation as bitcoin wallet.

File: translations/monero-core_zh-cn.ts [hunk 2]
```diff
@@ -2406,7 +2406,7 @@ Description: </source>
     <message>
         <location filename="../main.qml" line="965"/>
         <source>Program setup wizard</source>
-        <translation>程序设定精灵</translation>
+        <translation>程序设置</translation>
```
Comment (p1gd0g) [inline translations/monero-core_zh-cn.ts]: 程序设定向导。

Comment (zhizhongzhiwai) [inline translations/monero-core_zh-cn.ts]: 一样的意思。

**Comment by NanoAkron** [comment]
Hi @zhizhongzhiwai it would be excellent to give native Chinese speakers the option to use their own language for the wallet words. 

If you could make a separate PR to add the Chinese word list of 1626 common characters that would be excellent and would match with your intention to change the wallet language to 'Chinese' in this PR

File: lang/languages.xml (hunk 1 above)
Comment (Jaqueeee) [inline lang/languages.xml]: Before this PR can be merged you need to revert wallet_language back to "English" until there's a Chinese seed word list available.

**Comment by zhizhongzhiwai** [comment]
@NanoAkron 
after merge the commit , i will make a separate PR to add the Chinese word list.
question:  where the word list save to ? which format?

File: translations/monero-core_zh-cn.ts (hunk 2 above)
Comment (p1gd0g) [inline translations/monero-core_zh-cn.ts]: 俺觉得，不准确。当然俺只是建议，您无视俺就好。

**Comment by Jaqueeee** [comment]
@zhizhongzhiwai the word list should be added to the monero project repo.
You can have a look at this PR for reference:
https://github.com/monero-project/monero/pull/1918

Also please squash the commits in this PR into one.

**Comment by NanoAkron** [comment]
@Jaqueeee I've got a question about a Chinese word list - what value should `unique_prefix_length` take?

**Comment by zhizhongzhiwai** [comment]
i squashed commit.

**Comment by Jaqueeee** [comment]
No idea @nanoakron. I've never touched the seed language stuff. You better ask Moo or someone else in #monero-dev

**Comment by zhizhongzhiwai** [comment]
@p1gd0g are you from china?
i'm not c++ programmer. and i don't know how to make chinese words seed.
can you help?

**Comment by p1gd0g** [comment]
@zhizhongzhiwai 也许是这个？https://github.com/monero-project/monero/tree/master/src/mnemonics

**Comment by NanoAkron** [comment]
@zhizhongzhiwai and @p1gd0g Imagine you were typing in Chinese words from a word list to restore your wallet. Do you type them in roman alphabet characters (Shui = water) or do you type them in a different way?

**Comment by NanoAkron** [comment]
@zhizhongzhiwai @p1gd0g Please check out https://github.com/monero-project/monero/pull/1980

**Pull request merged**